Cargo.toml

```diff
@@ -13,7 +13,8 @@ edition = "2018"
 [dependencies]
 base64 = "0.10"
 chrono = "0.4"
-hyper = "0.10.2"
+hyper = "0.12"
+hyper-tls = "0.3"
```

Comment:
What Hyper TLS/HTTPS lib is the project heading towards?

Reply:
to be honest, I've lost orientation on which there are. Are you favoring one?